core/router/default.go

```diff
@@ -23,7 +23,7 @@ func Default(functions []config.Function) Router {
 }
 
 // Route get route from metadata.
-func (r *DefaultRouter) Route(metadata metadata.Metadata) Route {
+func (r *DefaultRouter) Route(metadata metadata.M) Route {
```

Comment:
Seems `Route()` ignored metadata here, shall we remove the parameter?

Reply:
We can't ignore `metadata` here because `DefaultRouter` implements the Router interface, which has a function `Route(metadata metadata.M) Route`.